README.md

```diff
@@ -1,21 +1,21 @@
 # thumbor_mongodb
 Thumbor MongoDB loader with GridFS support
 
-This project is based on https://github.com/dhardy92/thumbor_mongodb which is no longer maintained and hasn't support for GridFS.
+This project is based on https://github.com/timich/thumbor_mongodb which is no longer maintained and hasn't support for GridFS.
 This is my first time programming in python, so any PRs are welcome :)
 
+Wrok with tc_mongodb on https://github.com/thumbor-community/mongodb
+
 ```
 LOADER = 'thumbor_mongodb.loader' #loader calling
-MONGO_STORAGE_SERVER_HOST = 'localhost' #host
-MONGO_STORAGE_SERVER_PORT = 27017 #port
-MONGO_STORAGE_READ_PREFERENCE = 'primaryPreferred' #for from slave in replicaSet put there 'secondaryPreferred'
```

Comment:
this setting should be here because if you have master-slave config, thumbor should read from master

Reply:
I ask for time, I'm not good at English. my time GMT+7 tomorrow will me edit.

Reply:
I am GMT +1 😃  ok, but I will have time on Sunday, around 3PM GMT+1